pom.xml

```diff
@@ -9,13 +9,13 @@
 	<packaging>jar</packaging>
 	<groupId>org.webjars</groupId>
 	<artifactId>oscar</artifactId>
-	<version>15.0-SNAPSHOT</version>
+	<version>15.0.1-RC1</version>
```

Comment:
I think I must be misunderstanding something in the process.  My assumption was that I'd issue a pull request with the POM in the correct state in which to make a release from.  That is, both the upstream.url and version are set as to make a tag from it.

If I change the POM to a snapshot, then the upstream.url would also be reverted to pointing to a "floating" snapshot to have continuity within the project model.

Can you please clarify?